google/cloud/aiplatform/training_jobs.py

```diff
@@ -66,6 +67,289 @@
 )
 
 
+class _TrainingJob(ABC, base.AiPlatformResourceNoun):
```

Comment:
You shouldn't need to inherit `ABC` here because there are unimplemented `abstractmethods` in this class.